Sources/NIOPosix/Bootstrap.swift

```diff
@@ -253,6 +255,26 @@ public final class ServerBootstrap {
         return self
     }
 
+    /// Enables multi-path TCP support.
+    ///
+    /// This option is only supported on some systems, and will lead to bind
+    /// failing if the system does not support it. Users are recommended to
+    /// only enable this in response to configuration or feature detection.
+    ///
+    /// - parameters:
+    ///     - value: Whether to enable MPTCP or not.
+    public func enableMPTCP(_ value: Bool) -> Self {
+        self.enableMPTCP = value
+
+        // This is a temporary workaround until we get some stable Linux kernel
+        // versions that support TCP_NODELAY and MPTCP.
+        if value {
+            self._serverChannelOptions.remove(key: ChannelOptions.tcpOption(.tcp_nodelay))
+        }
```

Comment:
Will we be able to detect the linux kernel version at runtime in the future to make this a non-breaking change?

Reply:
Yes, in principle we can do this.